standalone/OpenMod.Standalone/StandaloneHost.cs

```diff
@@ -71,6 +72,12 @@ public Task ShutdownAsync()
 
         private void OnCommandExecute(string commandline)
         {
+            var trimmedCommand = string.Concat(commandline.Where(c => c.Equals(' ') || !char.IsWhiteSpace(c)));
```

Comment:
This check seems wrong, why is unicode space treated different than normal space?